### PR TITLE
Add BoxBoy!

### PR DESCRIPTION
**Pull request by Nyargleblargle**
Adds BoxBoy!.

*Automatically generated by PR builder*

### CODE REVIEW

File: boxboy/game.dat [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title = "BoxBoy!"
+description = "Puzzle Platformer developed by HAL Laboratory and published by Nintendo"
```
Comment (Schplee) [inline boxboy/game.dat]: Add a couple more lines from the wikipedia blurb, since this description doesn't include much.

**Comment by Nyargleblargle** [comment]
How would I edit the background color?

**Comment by Schplee** [comment]
@Nyargleblargle Please refer to our contributions guide: https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#screenshots-1

Also, please create page, following the instructions provided in the contributions guide: https://github.com/citra-emu/citra-games-wiki/wiki

File: boxboy/game.dat [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title = "0004000000154500"
+region = "usa"
+release_date = "2015-01-14"
+
+[[testcases]]
+title = "0004000000154500"
+compatibility = "0"
+date = "2017-07-19"
+version = "HEAD-2f625f2"
+cpu = "Intel Core i7"
```
Comment (TheKoopaKingdom) [inline boxboy/game.dat]: This could be more specific.

**Comment by chris062689** [comment]
@Nyargleblargle Has there been an update to this? Thanks.

**Comment by Schplee** [comment]
PR author looks to have abandoned it. Closing.

**Pull request closed without merging**